src/api/app/controllers/webui/request_controller.rb

```diff
@@ -29,12 +29,13 @@ class Webui::RequestController < Webui::WebuiController
                                   if: -> { Flipper.enabled?(:request_show_redesign, User.session) }
   before_action :check_beta_user_redirect, only: %i[build_results rpm_lint changes mentioned_issues]
   before_action :redirect_to_tasks, only: [:index], unless: -> { Flipper.enabled?(:request_index, User.session) }
-  before_action :set_requests, :set_filter_involvement, :set_filter_state, :filter_requests, :set_selected_filter, only: [:index], if: -> { Flipper.enabled?(:request_index, User.session) }
+  before_action :set_requests, :set_filter_involvement, :set_filter_state, :set_filter_creators, :filter_requests, :set_selected_filter, only: [:index], if: -> { Flipper.enabled?(:request_index, User.session) }
 
   after_action :verify_authorized, only: [:create]
 
   def index
     @bs_requests = @bs_requests.order('number DESC').page(params[:page])
+    @bs_requests_creators = @bs_requests.distinct.pluck(:creator)
```

Comment:
If I’m not mistaken, this will retrieve the creators of the requests of the current page.

I'd prefer to retrieve the creators of all the requests, not only those of the request shown in the current page.

Reply:
but whats the point? we only have those requests available. Why should I allow to filter against creators that are not available in this scope? Nothing will show up if I filter against a creator where no request is available in this scope

Reply:
Let me correct myself: I'd prefer to retrieve the creators of all the requests of the current scope, not only those of the requests shown in the current page.

Reply:
ah alright, hmm let me check